src/example-relay/next.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = withCustomBabelConfigFile(
       // @x-shipit-enable: 'babel.config.js',
       '.babelrc', // @x-shipit-disable
     ),
-    transpileModules: ['@adeira'],
+    // @kiwicom/orbit-components is using `@adeira/js`, and needs to be  tranpiled as well because of this
+    transpileModules: ['@adeira', '@kiwicom'],
```

Comment:
I don't understand this. Could you explain it better what's the problem? This should not be the case.

Reply:
in `@kiwicom/orbit-components`, there is `var _js = require('@adeira/js')`; Then when starting example relay, we get error because `export is undefined`. It is mapping this to our un-transpiled source code.

Reply:
So they are bundling untranspiled `adeira/js`? 🤔 How? I still don't understand. I guess I will have to check it closely.

Reply:
Try starting `example relay` locally.

> So they are bundling untranspiled adeira/js?

I don't think that is how it works. They transpile their code own code, and we download their dependencies. Then our repo will not download `@adeira/js` since it is an internal workspace. And requiring it from `@kiwicom/orbit` will point it to our untranspiled workspace